diff --git a/ml_drift_delegate/delegate/shared_memory_manager/BUILD b/ml_drift_delegate/delegate/shared_memory_manager/BUILD
index b9df121f2..304098d14 100644
--- a/ml_drift_delegate/delegate/shared_memory_manager/BUILD
+++ b/ml_drift_delegate/delegate/shared_memory_manager/BUILD
@@ -223,7 +223,7 @@ cc_library(
         "//ml_drift_delegate/tflite:shared_const_tensor_map",
         "//tflite/c:common",
         "//tflite/core/c:private_common",
-        "//third_party/odml/litert/weight_loader:external_weight_loader",
+        "//weight_loader:external_weight_loader",
         "@com_google_absl//absl/status",
         "@com_google_absl//absl/status:statusor",
         "@ml_drift//ml_drift/common:executor",
@@ -336,7 +336,7 @@ objc_library(
         "//litert/c/internal:litert_runtime_context_header",
         "//ml_drift_delegate/delegate:unowned_tensor_desc",
         "//ml_drift_delegate/delegate/serialization_weight_cache",
-        "//third_party/odml/litert/weight_loader:external_weight_loader",
+        "//weight_loader:external_weight_loader",
         "//tflite/c:common",
         "//tflite/core/c:private_common",
     ],
