diff --git a/ml_drift/common/gpu_info.h b/ml_drift/common/gpu_info.h
index 18b5759..bd8aa8e 100644
--- a/ml_drift/common/gpu_info.h
+++ b/ml_drift/common/gpu_info.h
@@ -633,10 +633,10 @@ struct OpenGlInfo {
 
 struct VulkanInfo {
   std::string vendor_name;
-  uint32_t api_version = -1;
-  uint32_t api_version_major = -1;
-  uint32_t api_version_minor = -1;
-  uint32_t api_version_patch = -1;
+  uint32_t api_version = UINT32_MAX;
+  uint32_t api_version_major = UINT32_MAX;
+  uint32_t api_version_minor = UINT32_MAX;
+  uint32_t api_version_patch = UINT32_MAX;
 
   uint32_t max_per_stage_descriptor_sampled_images = 0;
   uint32_t max_per_stage_descriptor_storage_images = 0;
